server/api/test/index.js

```diff
@@ -117,4 +117,24 @@ describe( 'api', function() {
 			.send( { username: 'validuser', password: 'validpassword' } )
 			.expect( 408, response, done );
 	} );
+
+	maybeIt( 'should return a 400 push verification request if the server supports it', function( done ) {
```

Comment:
It [runs the tests only in the 'desktop' environment](https://github.com/gyoshev/wp-calypso/blob/067035345cb3aa431f0fa089b4a6c1e11e8b15c2/server/api/test/index.js#L48-L51). Maybe `onDesktopIt` is a more appropriate name. I believe this is in place because the `/oauth` end-point is only available when OAuth is enabled via the config, i.e. on desktop. Can be changed if the methods are tested directly.